Code/Editor/RenderViewport.h

```diff
@@ -51,599 +51,4 @@ namespace AzToolsFramework
     class ManipulatorManager;
 }
 
```

Comment:
update: I think we should just delete `RenderViewprt.cpp` instead of leaving it empty

Reply:
Same for the .h too if at all possible